pkg/storage/unified/sql/backend.go

```diff
@@ -50,26 +51,29 @@ func NewBackend(opts BackendOptions) (Backend, error) {
 	}
 	ctx, cancel := context.WithCancel(context.Background())
 
-	pollingInterval := opts.PollingInterval
-	if pollingInterval == 0 {
-		pollingInterval = defaultPollingInterval
+	if opts.PollingInterval == 0 {
+		opts.PollingInterval = defaultPollingInterval
 	}
 	if opts.WatchBufferSize == 0 {
 		opts.WatchBufferSize = defaultWatchBufferSize
 	}
 	return &backend{
+		isHA:            opts.IsHA,
```

Comment:
Can we help out the little guy?
```suggestion
		isHA:            opts.IsHA && !SQLIte,
```

Reply:
nevermind, the isHA input already has this specified